karma.conf.js

```diff
@@ -3,7 +3,7 @@ process.env.CHROME_BIN = require('puppeteer').executablePath()
 module.exports = function (config) {
   config.set({
     frameworks: ['mocha', 'chai'],
-    files: ['test/**/*.js'],
+    files: ['test/*.js'],
```

Comment:
this will not run tests if they are in a subfolder, I think we should keep the same behavior as before

Reply:
ok, done